.dockerignore

```diff
@@ -1,10 +1,2 @@
 **
-!/api
-!/controllers
-!/internal
-!/redskyapi
-!/redskyctl
-!/go.mod
-!/go.sum
-!/main.go
```

Comment:
Not sure I'm following this change?

Reply:
We do not build the Go code inside the Docker images anymore, there is no point in copying the source in on the Docker context.